all.yml

```diff
@@ -1,6 +1,7 @@
 aws_ami:
 aws_autoscaling_group:
 aws_cloudformation_stack:
+aws_cloudtrail_trail:
```

Comment:
FYI, we use Terraform resource types as a resource type identifier. I assume that you mean the `aws_cloudtrail` resource? See: https://www.terraform.io/docs/providers/aws/r/cloudtrail.html

Reply:
Got it; apologies for having to hand-hold me through this pull request.